sdk/monitor/Azure.Monitor.Query/src/autorest.md

```diff
@@ -12,7 +12,6 @@ input-file:
 generation1-convenience-client: true
 modelerfour:
     lenient-model-deduplication: true
-    seal-single-value-enum-by-default: true
```

Comment:
Can you mention in the PR description that this was not necessary and that there were no changes after regenerating?